### ISSUE TITLE
serviceUrl defined in Constructor of MessageBirdService not being used 

### ISSUE THREAD

**Issue by HaraldWalker** [messagebird/java-rest-api#17]
Constructor of MessageBirdService allows to set the serviceUrl but the value is never being set and used. It will always use the hardcoded "https://rest.messagebird.com" as service url. 

```java
    public MessageBirdServiceImpl(final String accessKey, final String serviceUrl) {
        if (accessKey == null) {
            throw new IllegalArgumentException(ACCESS_KEY_MUST_BE_SPECIFIED);
        }
        if (serviceUrl == null || serviceUrl.length() == 0) {
            throw new IllegalArgumentException(SERVICE_URL_MUST_BE_SPECIFIED);
        }
        this.accessKey = accessKey;
    }
```

Not a big issue, although it allowed us to test the integration without actually using the MessageBird API.

**Comment by Ruudieboy** [comment]
Would be great if this can be merged and release to maven. We need this functionality.

**Comment by samwierema** [comment]
Merged and published as version 1.3.0 on Maven.

Thanks @HaraldWalker!
